### PR TITLE
CSP-1176 Upgrade das-employer-aan-web to .NET8

### PR DESCRIPTION
**Pull request by ManishAgarwalDfE**

### CODE REVIEW

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.Aan.SharedUi/Views/Error/ErrorInService.cshtml [hunk 1]
```diff
@@ -1,8 +1,11 @@
-﻿﻿<div class="govuk-grid-row">
+﻿@using SFA.DAS.Aan.SharedUi.Models;
+@model ErrorViewModel;
+
+﻿<div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
         <p class="govuk-body">
-            Refresh your browser, or return to the <a asp-action="Index" asp-controller="home">apprentice ambassador network home</a>
+            Refresh your browser, or return to the <a class="govuk-link" href="@Model.HomePageUrl">@Model.HomePageUrlText</a>
```
Comment (DevenKShah) [inline src/SFA.DAS.Aan.SharedUi/Views/Error/ErrorInService.cshtml:8]: What was the issue with this implementation, I can see that employer has a Home controller with Index method?

Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Aan.SharedUi/Views/Error/ErrorInService.cshtml:8]: yes agreed employer web has a home controller with index method but 1. it requires "employerAccountId" which is not used for apprentice web so created dynamic url for home page and also 2. the url text doesn't look ok for employer web which is "apprentice ambassador network home" so made it dynamic to change it as per portal.

Comment (DevenKShah) [reply src/SFA.DAS.Aan.SharedUi/Views/Error/ErrorInService.cshtml:8]: Fair point for home url. What do you reckon the text should  be for employer?

Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Aan.SharedUi/Views/Error/ErrorInService.cshtml]: Reverted

**Review by DevenKShah — DISMISSED** [review]

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**